client/src/components/profile-card.scss

```diff
@@ -0,0 +1,33 @@
+.profile-card.ant-card {
+    border-bottom: 2px solid #aeb8c7;
+    text-align: center;
+    position: sticky;
+    top: 50px;
+}
+
+#github1 {
```

Comment:
SASS supports nesting to avoid global style conflict

You should move everything below under the scope of ‘.profile-card’